README.md

```diff
@@ -44,6 +44,10 @@ You can do so by creating a `settings.json` in the root folder of the dub-regist
 
 It's recommended to create a separate account for the DUB registry GitHub authentication. Equally, if no GitLab packages are used in your local repository, no GitLab authentication is needed.
 
+It's absolutely recommended to create a personal access token without any extra permissions for your GitHub account instead of entering your password plain text into the settings file. You can generate an access token at https://github.com/settings/tokens (Settings -> Developer Settings -> Personal access tokens)
+
+Note that previous versions of this README stated to write down the password in plain text. This should be considered insecure as special characters in your password such as '@' could expose following parts of the password to the users.
```

Comment:
Didn't I ask for this paragraph to be removed as (1) it's unlikely that actually happened, (2) it's a separate account anyhow and (3) this sounds bad for future readers.

Reply:
From what I had before, I removed most of it but kept the information about that it is now using a token instead of password for people who have read the README before. I also thought just having a small notice would be good information for those who are searching for why they even put in a password in the first place and can't remember why they did it.